ui/StatusQ/tests/src/TestHelpers/testmodel.h

```diff
@@ -0,0 +1,22 @@
+#include <QAbstractListModel>
+
+class TestModel : public QAbstractListModel {
+
+public:
+    explicit TestModel(QList<QPair<QString, QVariantList>> data);
```

Comment:
```suggestion
    explicit TestModel(const QList<QPair<QString, QVariantList>>& data);
```

Reply:
In all cases those arguments are moved. So in all cases ctors/methods copy lvalue arguments, but move rvalue arguments. Therefore there is no performance overhead bc of the signatures.

Reply:
Alright... it's just not very common practice in Qt code to use move semantics :)